src/test/e2e/PageObjects/IdamSignin.page.ts

```diff
@@ -0,0 +1,36 @@
+import { SubscriptionManagementPage } from './SubscriptionManagement.page';
+
+const helpers = require('../Helpers/Selectors');
+
+export class IdamSigninPage {
+
+  open (path): Promise<string> {
+    return browser.url(path);
+  }
+
+  async getPageTitle(): Promise<string> {
+    $(helpers.CommonPageTitle).catch(() => {
+      console.log(`${helpers.CommonPageTitle} not found`);
+    });
+
+    return $(helpers.CommonPageTitle).getText();
+  }
+
+  async selectIdam(text): Promise<void> {
+    $(helpers.IdamSiginInput).catch(() => {
+      console.log(`${helpers.IdamSiginInput} not found`);
+    });
+    const selectInput = await $(helpers.IdamSiginInput);
+    selectInput.selectByVisibleText(text);
+  }
+
+  async clickContinue(): Promise<SubscriptionManagementPage> {
```

Comment:
Is user supposed to be taken to Subscription Management page or (external) auth page?

Reply:
user should be taken to an external page. However BA still didn't put it in the ticket